src/cclib/io/filewriter.py

```diff
@@ -44,7 +48,10 @@ def __init__(self, ccdata, jobfilename=None, terse=False,
         self.terse = terse
 
         self.pt = PeriodicTable()
-        self.elements = [self.pt.element[Z] for Z in self.ccdata.atomnos]
+        if hasattr(self.ccdata, 'atomnos'):
```

Comment:
While that might be true for Molden, is it necessarily true for all future writers?